engines/io_uring.c

```diff
@@ -82,11 +82,14 @@ struct ioring_data {
 	struct cmdprio cmdprio;
 
 	struct nvme_dsm *dsm;
+	unsigned int cdw12_flags[DDIR_RWDIR_CNT];
```

Comment:
Please change this to `uint32_t` just in case this is ever compiled one day on a platform where `int` is not 4 bytes.

Reply:
Updated, Thanks !